src/preact/slot.js

```diff
@@ -70,11 +77,53 @@ export function useSlotContext(ref) {
       Slot,
       /** @type {!./core/loading-instructions.Loading} */ (context.loading)
     );
+
+    if (!context.playable) {
+      execute(slot, pauseAll);
+    }
+
     return () => {
       removeProp(slot, CanRender, Slot);
       removeProp(slot, CanPlay, Slot);
       removeProp(slot, LoadingProp, Slot);
       rediscoverChildren(slot);
     };
   }, [ref, context]);
+
+  // Unmount and unmount. Keep it at the bottom because it's much better to
```

Comment:
Mount and unmount?

Reply:
https://github.com/ampproject/amphtml/pull/33502